main/springboot/build.gradle

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
> Have you thought of the tradeoffs between putting this in it's own top-level module of springboot vs a sub-module under main?

> To me it seems like it would be more natural in it's own top-level module (i.e., it's not really a part of the "main" application), but you may have thought of reasons why that shouldn't exist. Also, it seems a little strange to have a "main/cli" module that actually includes a lot more than the CLI.


During my experiment for some reason having springboot as a top-level module was causing issue - however not it seems to work fine. So, per your suggestion I'll refactor.

Reply:
> Also, no matter what, the BUILD.md file needs to be updated to explain how many JARs are generated when you execute ./gradlew clean build and where they are located and what they do, and how to build the sub-modules if you only want to build the CLI (e.g., if building springboot takes longer).

Will do.